### DISCUSSION TITLE
Feature Request: "Mark page as read" in MDN Learn web development course

### DISCUSSION THREAD

**Discussion by dawidl022** [mdn/mdn-community#557] [Content] [answered]
Hello, I have a feature request: It would be really nice to have an option to "mark a page as read" in the MDN Learn web development course.

This way it would be possible to easily see what one has already studied, and what is left to study. A basic kind of progress tracking system.

**Comment by wbamberg** [comment]
FWIW this would be very implementable as a browser extension.

**Comment by Rumyra** [comment, marked as answer]
Thanks @dawidl022 - this is actually a feature we are currently considering ☺️

**Comment by pepelsbey** [comment]
Closing as answered/settled.
